src/crawler.ts

```diff
@@ -145,6 +145,10 @@ export class ResolveCrawler implements Types.ICrawler {
           this.pointerGraph.addNode(targetPointer);
         }
 
+        const srn = `${this._runner.baseUri.toString()}${targetPointer}`;
```

Comment:
SRN would be a new concept in this library, and this is less of an SRN and more of just the full uri right?

Reply:
Agreed, SRN is specific to stoplight. And it shouldn’t be specific to the resolver. But the idea for using the srn, was to make it easier for the platform to build a go to ref url. If that makes sense. By no means is this ready to be merged, it’s too specific to Stoplight.

Reply:
I am going to change it to the full URLs. We can get the SRNs on the front end.